### PR TITLE
Redirect after POST when returning from GSSP

### PR DESCRIPTION
**Pull request by jorissteyn**
The ACS location of GSSP should never show a page without first
redirecting. This way, the user can refresh the page or change the
language before trying again.

See: https://www.pivotaltracker.com/story/show/157843953

### CODE REVIEW

File: src/Surfnet/StepupSelfService/SelfServiceBundle/Controller/Registration/GssfController.php [hunk 1]
```diff
@@ -105,8 +113,8 @@ public function consumeAssertionAction(Request $httpRequest, $provider)
                 sprintf('Could not process received Response, error: "%s"', $exception->getMessage())
             );
 
-            return $this->renderInitiateForm(
-                $provider->getName(),
+            return $this->redirectToInitiationForm(
+                $provider,
                 ['authenticationFailed' => true]
```
Comment (MKodde) [inline src/Surfnet/StepupSelfService/SelfServiceBundle/Controller/Registration/GssfController.php]: Why pass the provider alongside the options when all you do in the `redirectToInitiationForm` is: adding the provider name to the options?

File: src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/config/routing.yml [hunk 2]
```diff
@@ -63,7 +63,10 @@ ss_registration_sms_prove_possession:
 ss_registration_gssf_initiate:
     path:     /registration/gssf/{provider}/initiate
     methods:  [GET]
-    defaults: { _controller: SurfnetStepupSelfServiceSelfServiceBundle:Registration/Gssf:initiate }
+    defaults:
+        _controller: SurfnetStepupSelfServiceSelfServiceBundle:Registration/Gssf:initiate
+        authenticationFailed: false
```
Comment (MKodde) [inline src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/config/routing.yml]: Maybe add a doc block for the initiate action in the `GssfController` explaining the defaults that are set for the action and what their intention is.

Comment (jorissteyn) [reply src/Surfnet/StepupSelfService/SelfServiceBundle/Resources/config/routing.yml]: I've documented them in the controller action docblock.

File: src/Surfnet/StepupSelfService/SelfServiceBundle/Controller/Registration/GssfController.php (hunk 1 above)
Comment (jorissteyn) [reply src/Surfnet/StepupSelfService/SelfServiceBundle/Controller/Registration/GssfController.php]: We could pass a string, but then I can't be sure the string refers to a valid GSSP provider.

**Pull request merged**